native/packages/react-native-bpk-component-theming/stories.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  View,
+  Platform,
+  Picker,
+} from 'react-native';
+
+import { storiesOf } from '@storybook/react-native';
+import { withTheme } from 'theming';
+
+import BpkText from 'react-native-bpk-component-text';
+
+import BpkThemeProvider from './src/BpkThemeProvider';
+
+const tokens = Platform.OS === 'ios' ?
+  require('bpk-tokens/tokens/ios/base.react.native.common.js') :
+  require('bpk-tokens/tokens/android/base.react.native.common.js')
+;
+
+// TODO when a themeable component is created, import and use that
+// instead of making one here.
+const BpkThemeableText = withTheme((props) => {
+  const { theme, style: outerStyle, ...rest } = props;
+  const innerStyle = {
+    color: theme.textOnPrimaryColor,
+  };
+  return <BpkText {...rest} style={[outerStyle, innerStyle]} />;
+});
+
+class BpkThemePicker extends Component {
```

Comment:
I suspect we'll want to have the `BpkThemePicker` in all themeable component stories right? Might as well generalize it already

Reply:
Yeah good point - I'm not sure where's best for that to live though, what do you think?